# Type: test
# Subtype: T1
# English Summary: Rename test class RebalanceLargeTest to RebalanceLongTest
# Chinese Summary: 重命名测试类RebalanceLargeTest为RebalanceLongTest
# Entry Number: 18

# === Git Diff Content ===

diff --git a/test/long/voldemort/client/rebalance/RebalanceLongTest.java b/test/long/voldemort/client/rebalance/RebalanceLongTest.java
index 16d5b5091..eecba30bb 100644
--- a/test/long/voldemort/client/rebalance/RebalanceLongTest.java
+++ b/test/long/voldemort/client/rebalance/RebalanceLongTest.java
@@ -28,14 +28,14 @@ import voldemort.store.metadata.MetadataStore.VoldemortState;
  * 
  */
 @RunWith(Parameterized.class)
-public class RebalanceLargeTest extends AbstractRebalanceTest {
+public class RebalanceLongTest extends AbstractRebalanceTest {
 
     Map<Integer, VoldemortServer> serverMap = new HashMap<Integer, VoldemortServer>();
     private final boolean useNio;
     private final boolean useDonorBased;
     protected static int NUM_MANY_KEYS = 10100;
 
-    public RebalanceLargeTest(boolean useNio, boolean useDonorBased) {
+    public RebalanceLongTest(boolean useNio, boolean useDonorBased) {
         this.useNio = useNio;
         this.useDonorBased = useDonorBased;
     }
